bacchus_gazebo/launch/vineyard_demo.launch

```diff
@@ -27,7 +27,7 @@
 
 
   <!-- ############################## GAZEBO ############################## -->
-  <arg name="world_name" default="vineyard_stage0"/> <!-- Note: the world_name is with respect to GAZEBO_RESOURCE_PATH environmental variable -->
+  <arg name="world_name" default="vineyard_stage0_heightmap"/> <!-- Note: the world_name is with respect to GAZEBO_RESOURCE_PATH environmental variable -->
```

Comment:
Instead of having another identical launch file but the world model, we should only change the world model argument if required. I set the heightmap as our default world model now!